### PR TITLE
CM-922: How to Find the Latitude and Longitude of a Location in C#

### PR DESCRIPTION
**Pull request by elliezub**

### CODE REVIEW

**Review by elfico — CHANGES_REQUESTED** [review]

File: csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Models/CustomMapPoint.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using System.ComponentModel.DataAnnotations;
+namespace Models;
+
+public class CustomMapPoint
+{
+    [Required]
+    public double Latitude { get; set; }
+    public double Longitude { get; set; }
```
[the remark this answers is not in the document]
Comment (elfico) [reply csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Models/CustomMapPoint.cs:8]: @elliezub

File: csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/BenchmarkRunner/LocationServicesBenchmark.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+using BenchmarkDotNet.Attributes;
+using Services;
+using Microsoft.Extensions.Configuration;
+
+namespace BenchmarkRunner
+{
+    [MaxIterationCount(30)]
```
Comment (elfico) [inline csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/BenchmarkRunner/LocationServicesBenchmark.cs:7]: @elliezub 
Let's add an `Ordering` to the benchmarg

**Review by elfico — CHANGES_REQUESTED** [review]

File: csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/LatLongWithHttpClient.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+namespace Services;
+using System.Net.Http;
```
Comment (elfico) [inline csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/LatLongWithHttpClient.cs:2]: @elliezub 
We should remove unsed `using`s

File: csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/GoogleLocationService.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+using GoogleMaps.LocationServices;
+using Models;
```
Comment (elfico) [inline csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/GoogleLocationService.cs:2]: @elliezub 
We should also remove unused `usings` here

**Review by elfico — CHANGES_REQUESTED** [review]
@elliezub 
I am still getting errors when trying to run it. I renamed the solution and project and it worked so I think we should consider doing this. Find a sample error below:|
Screenshot_7

File: csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/GoogleLocationService.cs (hunk 4 above)
Comment (elliezub) [reply csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/GoogleLocationService.cs:2]: These two seem like they need to be here but I was able to remove using Services;

**Review by elfico — APPROVED** [review]

**Pull request merged**